DMPowerTools/DataModels/ActiveCreatures.cs

```diff
@@ -0,0 +1,18 @@
+﻿using DMPowerTools.Data;
+
+namespace DMPowerTools.DataModels
+{
+    public class ActiveCreature
```

Comment:
Given the name of this I would expect there to only be one active creature. Should this be called something else? InCombatCreature? InitiatedCreature? 🤷‍♂️. 

Also I don’t understand what the id is for here.